perceval/backends/core/groupsio.py

```diff
@@ -201,7 +201,12 @@ def fetch(self, from_date=None):
         if not os.path.exists(self.dirpath):
             os.makedirs(self.dirpath)
 
-        group_id = self.__find_group_id()
+        group_id, group_download_archive = self.__find_group_info()
+
+        if not group_download_archive:
+            msg = " Enable download archive permission for the group %s" % self.group_name
```

Comment:
I have a small dobut here, do you think it's better to report the error `Download archive permission disable for the group ...` or  suggest the solution `Enable download ..` ?

Minor comment, please remove the empty space before `Enable`

Reply:
Your version seems better. After all its an error message. Good Catch!!